install.sh

```diff
@@ -481,6 +481,20 @@ assert_usable_kernel() {
   fi
 }
 
+assert_kernel_has_seccomp() {
+  ret = prctl(PR_GET_SECCOMP, 0, 0, 0, 0);
```

Comment:
Hi @RedRudeBoy ,

Thanks for submitting this change!

I agree that it can be useful to test for seccomp existing.

However, I believe this change that you are suggested would introduce invalid bash syntax. It looks like you're writing C, but since `bash` is executing this script, I believe it would error-out with a syntax error.

Let me know what you think. Thank you again for submitting it!

I'll wait for your reply before testing the change.